content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -311,7 +311,7 @@ kubectl logs -f my-pod -c my-container              # stream pod container logs
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
 kubectl run nginx --image=nginx -n 
```

Comment:
aside: _this_ line should end with `\` to act as a continuation character, and I'd indent the next line to make that clear.